salt/orchestrate/edx/init.sls

```diff
@@ -32,7 +32,7 @@ generate_edx_cloud_map_file:
           consul-agent: {{ salt.boto_secgroup.get_group_id(
             'consul-agent-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
         subnetids: {{ subnet_ids }}
-        ami_id: {{ purposes.versions.ami_id }}
+        ami_id: {{ purposes['{}-draft'.format(PURPOSE_PREFIX)].ami_id }}
```

Comment:
Shouldn't this be {{ purposes['{}-draft'.format(PURPOSE_PREFIX)].versions.ami_id }}?